src/MSBuild.Conversion.Facts/WinUIFacts.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace MSBuild.Conversion.Facts
+{
+    /// <summary>
+    ///  A bunch of known values regarding WinUI2 projects.
+    /// </summary>
+    public static class WinUIFacts
+    {
+
+        /// <summary>
+        /// The core set of references WinUI projects use.
+        /// </summary>
+        /// <remarks>
+        /// NEED to update some of these
+        /// </remarks>
+        public static ImmutableArray<string> KnownWinUIReferences => ImmutableArray.Create(
+            "Microsoft.UI.Xaml",
+            "Windows.UI.Xaml",
+            "Microsoft.WinUI",
+            "Microsoft.NETCore.UniversalWindowsPlatform"
+        );
+
+        /// <summary>
+        /// Set of unecessary properties that are ok to remove
+        /// </summary>
+        public static ImmutableArray<string> UnnecessaryProperties => ImmutableArray.Create(
+            "DefaultLanguage",
+            "ProjectGuid", // Are these important? should they be converted instead?
+            "ProjectTypeGuids", //same as above
+            "SubType"
+
+        );
+
+        /// <summary>
+        /// Set of NuGet packages incompatible with WinUI3
+        /// </summary>
+        public static ImmutableArray<string> IncompatiblePackages => ImmutableArray.Create(
+            "Put.Bad.Pkg.Here"
+        );
+
+        //TODO Delete this
+        public static ImmutableArray<string> WinUIRefs => ImmutableArray.Create(
+            "Microsoft.UI.Xaml", 
+            "Microsoft.WinUI" 
+        );
+
+        public static ImmutableDictionary<string, string> ConvertiblePackages => ImmutableDictionary.CreateRange(new Dictionary<string, string>
+        {
+            { "Microsoft.UI.Xaml", "Microsoft.WinUI" },
+            { "Microsoft.Xaml.Behaviors.UWP", "Microsoft.Xaml.Behaviors.WinUI" },
+            { "Microsoft.Win2D.UWP", "Microsoft.Win2D.WinUI"}
+        });
+
+        public static ImmutableDictionary<string, string> PackageVersions => ImmutableDictionary.CreateRange(new Dictionary<string, string>
+        {
+            { "Microsoft.WinUI", "3.0.0-preview2.200713.0" },
+            { "Microsoft.Xaml.Behaviors.WinUI", "NeedVersion" },
+            { "Microsoft.Win2D.WinUI", "NeedsVersion"},
+            { "Microsoft.Toolkit", "2.6.6" }
```

Comment:
this doesn't seem right, they have a whole suite of packages.

@azchohfi can you provide a list of packages and versions from the WCT that that are supporting WinUI3?

Reply:
This is our MyGet feed:
https://dotnet.myget.org/feed/Packages/uwpcommunitytoolkit
Search for Microsoft.Toolkit, and anything that has an 8.x version (e.g. 8.0.0-preview2) is supporting WinUI3 with .Net5.

Reply:
This is a full list:
Microsoft.Toolkit.8.0.0-preview2
Microsoft.Toolkit.HighPerformance.8.0.0-preview2
Microsoft.Toolkit.Parsers.8.0.0-preview2
Microsoft.Toolkit.Services.8.0.0-preview2
Microsoft.Toolkit.Uwp.8.0.0-preview2
Microsoft.Toolkit.Uwp.Connectivity.8.0.0-preview2
Microsoft.Toolkit.Uwp.DeveloperTools.8.0.0-preview2
Microsoft.Toolkit.Uwp.Input.GazeInteraction.8.0.0-preview2
Microsoft.Toolkit.Uwp.Notifications.8.0.0-preview2
Microsoft.Toolkit.Uwp.Notifications.JavaScript.8.0.0-preview2
Microsoft.Toolkit.Uwp.PlatformSpecificAnalyzer.8.0.0-preview2
Microsoft.Toolkit.Uwp.UI.8.0.0-preview2
Microsoft.Toolkit.Uwp.UI.Animations.8.0.0-preview2
Microsoft.Toolkit.Uwp.UI.Controls.8.0.0-preview2
Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.8.0.0-preview2
Microsoft.Toolkit.Uwp.UI.Controls.Layout.8.0.0-preview2
Microsoft.Toolkit.Uwp.UI.Media.8.0.0-preview2

Reply:
Added these to WinUIFacts